examples/coap/coap-example-client/coap-example-observe-client.c

```diff
@@ -115,15 +113,15 @@ notification_callback(coap_observee_t *obs, void *notification,
  * Toggle the observation of the remote resource
  */
 void
-toggle_observation(void)
+toggle_observation(coap_endpoint_t* server_ep)
```

Comment:
Can the argument be `const`?

Formatting: the other pointers use the style `type *thing` rather than `type* thing`.